js.js

```
@@ -0,0 +1,52 @@
+}
+
+for (let i = 0; i < 100; i++) {
+    for (let j = 0; j < 100; j++) {
+        if (i * j === 5000) {
+            console.log("Found it!");
+        }
+    }
+}
+
+// Insecure input handling - possible XSS vulnerability
+document.getElementById("output").innerHTML = "User: " + window.location.search;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)